src/js/views/globals/patient-modal/patient-modal_views.js

```diff
@@ -280,11 +320,6 @@ const AddPatientModal = View.extend({
   onClickSave() {
     this.model.isValid({ preSave: true });
 
-    if (!isEmpty(this.model.validationError)) {
```

Comment:
The modal SaveView rerenders on model change now, so no need to check for errors on save